### PR TITLE
Add warning when moving private thread to open group

### PR DESCRIPTION
**Pull request by HSalmon**
[Trello card](https://trello.com/c/fh2ViWIi/831-add-warning-when-moving-a-private-discussion-to-a-only-public-discussions-group)

### CODE REVIEW

File: lineman/app/components/move_thread_form/move_thread_form.coffee [hunk 1]
```diff
@@ -12,3 +12,13 @@ angular.module('loomioApp').factory 'MoveThreadForm', ->
       flashSuccess: 'move_thread_form.messages.success'
       flashOptions:
         name: -> $scope.discussion.group().name
+
+    $scope.updateTarget = ->
+      $scope.targetGroup = Records.groups.find($scope.discussion.groupId)
+    $scope.updateTarget()
+
+    $scope.moveThread = ->
+      if $scope.discussion.private == true and $scope.targetGroup.privacyIsOpen()
```
Comment (gdpelican) [inline lineman/app/components/move_thread_form/move_thread_form.coffee]: No need for `== true` here

File: lineman/app/components/move_thread_form/move_thread_form.coffee [hunk 2]
```diff
@@ -12,3 +12,13 @@ angular.module('loomioApp').factory 'MoveThreadForm', ->
...
+        $scope.submit() if confirm($filter('translate')('move_thread_form.confirm_change_to_private_thread', groupName: $scope.targetGroup.name))
```
Comment (gdpelican) [inline lineman/app/components/move_thread_form/move_thread_form.coffee]: Check out `privacy_string_service.coffee` for an example of $translate.instant, which I think is a nicer way to handle this

**Comment by gdpelican** [comment]
Great work!

**Comment by HSalmon** [comment]
@gdpelican, how's that?

**Comment by gdpelican** [comment]
The best!

**Comment by HSalmon** [comment]
@robguthrie, @gdpelican, I've implemented James' feedback. I think this is ready to go - do you have any other concerns that need to be addressed?

**Comment by gdpelican** [comment]
It's great Hannah, I'd just like to hold off on merging stuff for the
moment until we get the e2ez up and running again; should be soon thanks to
the yeoman's work Rob's been up to, thanks Rob!

**Comment by robguthrie** [comment]
I've just updated this against master, with fixed e2es. I'll merge it soon. Great work, Thank you @HSalmon

**Pull request merged**